### PR TITLE
feature: TR-2486. Add new configuration option for review plugin

### PR DESCRIPTION
**Pull request by svoronuk**
Related to: https://oat-sa.atlassian.net/browse/TR-2486

Add option `reviewLayout` which allow select another variant of layout

You can test this in PR https://github.com/oat-sa/extension-tao-mext/pull/20

### CODE REVIEW

**Review by olga-kulish — COMMENTED** [review]
I think `views/css/new-test-runner.css` doesn't contain the latest version of `navigatorFizzy.scss`?  Please don't forget to update it afterwards

Screenshot_6
.

File: views/package-lock.json [hunk 1]
```diff
@@ -1,29 +1,8 @@
 {
   "name": "@oat-sa/tao-testqti",
   "version": "0.4.2",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```
[the remark this answers is not in the document]
Comment (svoronuk) [reply views/package-lock.json:4]: Yes. It is correct. Moving to version 2 should be made over all extensions/envs/... at once.

File: config/default/testRunner.conf.php [hunk 2]
```diff
@@ -389,6 +389,12 @@
              */
             'contentNavigatorType' => 'default'
         ],
+        /**
+         * The plugin renders review panel for test taker
+         */
+        'review' => [
+            'reviewLayout' => 'fizzy'
```
[the remark this answers is not in the document]
Comment (svoronuk) [reply config/default/testRunner.conf.php:396]: I don't know how it works, which i wrote in PR) I hope @jsconan can confirm/suggest something.

[the remark this answers is not in the document]
Comment (bziondik) [reply config/default/testRunner.conf.php:396]: default should be previous layout, new layout should be set in configuration by script/migration that will be executed in customer extension

**Review by bziondik — CHANGES_REQUESTED** [review]
Change default to old layout and use migrations for new

File: config/default/testRunner.conf.php (hunk 2 above)
[the remark this answers is not in the document]
Comment (svoronuk) [reply config/default/testRunner.conf.php]: Set default and create migration in https://github.com/oat-sa/extension-tao-mext/pull/20

**Review by bziondik — APPROVED** [review]
- [ ]  New code is covered by tests (if applicable)
- [ ]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [x]  New code is not subject to concurrency issues (if applicable)
- [ ]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

File: config/default/testRunner.conf.php [hunk 3]
```diff
@@ -389,6 +389,15 @@
...
         ],
+        /**
+         * The plugin renders review panel for test taker
+         * - `default`: the regular layout
+         * - `fizzy`: simplified layout aligned with NextGen styling
+         * @type string
+         */
+        'review' => [
+            'reviewLayout' => 'default'
+        ]
```
Comment (olga-kulish) [inline config/default/testRunner.conf.php:400]: Should be removed or commented?

**Review by olga-kulish — COMMENTED** [review]
Do you also need to update js bundles?

It should be done by release tool on merge github action, but I saw those files updated explicitly in other PRs like https://github.com/oat-sa/extension-tao-testqti/pull/2083/files or https://github.com/oat-sa/extension-tao-testqti/pull/2080/files
Though https://oatsa.slack.com/archives/C016PKDQPU5/p1635853090275200?thread_ts=1635497456.238800&cid=C016PKDQPU5

File: config/default/testRunner.conf.php (hunk 3 above)
Comment (svoronuk) [reply config/default/testRunner.conf.php:400]: Nope. This is new option. Unfortunately we do not have place to document new option, like `README.md`. As i see all properties are described in config files, so i did the same - added this "default" option with description.

Comment (olga-kulish) [reply config/default/testRunner.conf.php:400]: Sorry, my mistake - didn't notice you changed property value

**Review by olga-kulish — APPROVED** [review]
All is good. Js bundle may be needed for TAE. 
Still need to update `tao-test-runner-qti-fe` version in package.json

**Pull request merged**